README.md

```diff
@@ -185,3 +185,31 @@ Now that the ConfigMap has been created on your Kubernetes cluster, you can refe
 ```
 
 The chart will mount the content of the ConfigMap as a new `app-config.extra.yaml` file and automatically pass the extra configuration to your instance.
+
+### Configuring Chart PostgreSQL
+
+With the Backstage Helm Chart, it offers - as a subchart - a Bitnami PostgreSQL database. This can be enabled by switching `postgresql.enabled` to true (it is `true` by default). If switched on, the Helm Chart, on deployment, will automatically deploy a PostgreSQL instance and configure it with the credentials you specify. There are multiple ways of doing this that will be detailed below.
+
+#### Automatic Database Credential Creation
+
+This is the easiest of the configuration options. Here, the credentials for both the Admin and Database user will be automatically generated and put into a Kubernetes secret. This then will be automatically used by Backstage. For this, ensure the following happens:
+
+- Keep `postgresql.auth.existingSecret` & `postgresql.auth.password` empty.
+
+#### Specifying Password for PostgreSQL to Use
+
+Here, you can specify the password that you want PostgreSQL to use for it's Database User (The user that Backstage will use to connect to the database). For this, ensure the following happens:
+
+- Keep the `postgresql.auth.existingSecret` empty.
+- Set the `postgresql.auth.password` to the value that you want the User password to be.
+
+> **_NOTE:_** Be careful and mindful that the value you provide here is done in a secure way.
```

Comment:
```suggestion
> **_NOTE:_** Be careful that you provide this value securely.
```

Reply:
Do we need this note? Isn't `postgresql.auth.password` going to be unsecure always?

Reply:
It's worth keeping in. The industry is adopting GitOps more and more, so declaring the Backstage Helm Deployment in a [`HelmRelease`](https://fluxcd.io/docs/components/helm/helmreleases/) (if using FluxCD) allows for GitOps agents to reconcile the state of the cluster with what is configured in git. Together with this, you are able to put secrets into Git by using things like Bitnami Sealed Secrets or SOPS. So for those who use these methods, they would be able to pass the `password` field into the subchart securely using SOPS.